core/runner_actor.py

```diff
@@ -0,0 +1,102 @@
+# Add the parent directory to the PATH to allow imports.
+import os, sys, inspect
+currentdir = os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe())))
+parentdir = os.path.dirname(currentdir)
+sys.path.insert(0,parentdir)
+import logging
+import random
+import uuid
+import ray
+
+from core.runner import DMLRunner
+from custom.keras import model_from_serialized, get_optimizer
+from data.iterators import count_datapoints
+from data.iterators import create_train_dataset_iterator
+from data.iterators import create_test_dataset_iterator
+from core.utils.keras import train_keras_model, validate_keras_model
+from core.utils.keras import serialize_weights
+
```

Comment:
Extra line here. Our style guide is 2 lines between imports and code.

Reply:
Done.